flake.nix

```diff
@@ -39,13 +45,15 @@
       devShells."aarch64-darwin".autorelease-macos =
         import ./scripts/macos-shell.nix { pkgs = pkgs-darwin; };
 
+      overlay = final: prev: nixpkgs.lib.composeManyExtensions [
+        ocaml-overlay
+        (final: prev: { inherit (inputs) serokell-nix; })
```

Comment:
that adds all of serokell-nix flake into the overlay, shouldnt it be `[ ocaml-overlay ]++ (mapAttrsToList (_: v: v) serokell-nix.overalays)`

Reply:
The goal of this overlay is to provide types from `serokell.nix`, that aren't exposed in the overlay, AFAICS